src/components/PageBanner/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PageBanner.module.scss';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import { isFunction } from 'lodash';
+
+const PageBanner = props => {
+  const { children, wrapClass, className, dismissable } = props;
```

Comment:
The one thing I'd change here - is give this a visible property. Likely the onClose function that gets called in the parent would switch visible to false. That's the general pattern we've had with our components (see modal as an example)

Reply:
Okay so you'd rather the management of open state be on the parent? Right now the internal onClose performs the animation. I could move the animation to componentDidUpdate if the `visible` property changes.

Reply:
Yeah that would work - you could also use the transition component as a wrapper and have the children exist or not exist depending on the prop existing.  Either way should work fine though and I don't have a major preference here.

Reply:
If you explore this method - you can see in modal/index.js how it works as an example...